e2e/helpers.js

```diff
@@ -109,6 +113,8 @@ export function terminateApp( deviceId, bundleId ) {
 }
 
 export async function iNatE2eAfterEach( device ) {
+  await resetUserForTesting();
```

Comment:
Is it redundant resetting before and after each test? Not a big deal though if `resetUserForTesting` is a quick operation